docs/requirements/TestRequirements.md

```diff
@@ -101,7 +101,9 @@ Tests to implement
 |R82|Test QNAME Case insensitivity                            |[NAMESERVER](../specifications/tests/Nameserver-TP/nameserver08.md)|
 |R83|Consistency between glue and authoritative data          |[CONSISTENCY](../specifications/tests/Consistency-TP/consistency05.md)|
 |R84|Test for DNSSEC Algorithm Completeness (DS->DNSKEY->RRSIG)|[DNSSEC](../specifications/tests/DNSSEC-TP/dnssec12.md)|
-
+|R86|Test for undefined EDNS OPTION-CODE |[NAMESERVER](../specifications/tests/Nameserver-TP/nameserver11.md)|
+|R86|Test for unknown EDNS flags
+|[NAMESERVER](../specifications/tests/Nameserver-TP/nameserver12.md)|
```

Comment:
* R86 is already taken in zonemaster#651.
* You have the same requirement number twice.
* There a linebreak that should not be there.